src/components/Dropdown.js

```diff
@@ -0,0 +1,270 @@
+import React from 'react'
+import CreatableSelect from 'react-select/creatable'
+import ReactSelect from 'react-select'
+import styled, { withTheme, css } from 'styled-components'
+import { components } from 'react-select'
+import customCursor from '../assets/custom-cursor.png'
+
+const sharedStyle = css`
+  .react-select__control {
+    background-color: transparent;
+    margin: 2em 0;
+    border: 2px solid;
+    border-radius: 7px;
+    box-shadow: none;
+    max-width: ${p => (p.isSearchable ? `387px` : `287px`)};
+    height: 48px;
+    border-color: ${p =>
+      p.isValid ? p.theme.colors.dropdown.neutral : p.theme.colors.dropdown.error};
+    padding-right: 17px;
+    padding-left: 7px;
+    flex-direction: ${p => p.isSearchable && `row-reverse`};
+    cursor: url(${customCursor}), auto;
+  }
+
+  .react-select__control:hover,
+  .react-select__control--is-focused {
+    border-color: ${p => p.theme.colors.dropdown.background};
+  }
+
+  .react-select__option {
+    color: ${p => p.theme.colors.background};
+    overflow-wrap: break-word;
+  }
+
+  .react-select__menu {
+    background-color: ${p => p.theme.colors.dropdown.background};
+    border-radius: 7px;
+    margin-top: 0px;
+    max-width: ${p => (p.isSearchable ? `387px` : `287px`)};
```

Comment:
I know you followed the exact dimensions of the components on figma, but since the no-search dropdown width should match that of the inputs, I think you could do something similar to what @acchiang did for the textinputs: 
```
const inputSize = {
  default: '300px',
  medium: '400px',
  large: '600px',
}
```
and the set max-width to something like
```
max-width: ${p => (p.isSearchable ? inputSize.medium : inputSize.default)};
```
Same with line 15.

<img width="436" alt="Screen Shot 2020-11-21 at 12 03 30 AM" src="https://user-images.githubusercontent.com/38872354/99871138-ffaee580-2b8c-11eb-9a20-aec4225f7d3c.png">